packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp

```diff
@@ -142,7 +142,65 @@ void StepperIMEX_RK<Scalar>::setTableaus(std::string stepperType,
     this->setStepperType("IMEX RK 1st order");
     this->setOrder(1);
 
-  } else if (stepperType == "IMEX RK SSP2" || stepperType == "SSP2_222" ) {
+  } else if ( stepperType == "SSP1_111" ) 
```

Comment:
@sconde Could you add some more comments from a time integrator perspective as to the differences between this method and the IMEX RK 1st order method above?